src/chatlog/chatmessage.cpp

```diff
@@ -209,8 +210,10 @@ QString ChatMessage::detectMarkdown(const QString &str)
 
             QString htmledSnippet;
 
-            // Check for surrounding spaces and/or beginning/end of line
-            if (!((snipCheck.startsWith(' ') || offset == 0) && (snipCheck.endsWith(' ') || offset + snipCheck.trimmed().length() == out.length())))
+            qDebug() << "snip: " << snipCheck << " off+leng: " << offset + snipCheck.trimmed().length() << " out.len: " << out.length() << " offset: " << offset;
+
+            // Check for surrounding spaces, newlines and/or beginning/end of line
+            if (!(((snipCheck.startsWith(' ') || snipCheck.startsWith('>')) || offset == 0) || ((snipCheck.endsWith(' ') || snipCheck.endsWith('<')) && offset + snipCheck.trimmed().length() == out.length())))
```

Comment:
After trying to convert this to a singular QRegExp for the past three hours I can say this is the easier route.